yarn.lock

```diff
@@ -132,6 +132,13 @@
   dependencies:
     "@babel/highlight" "^7.16.7"
 
+"@babel/code-frame@^7.12.13":
```

Comment:
@vojtechjelinek I'm a bit confused about the changes in this file. I had initially, incorrectly used npm install to add the new package but later on I reset the contents of this file and used yarn install to regenerate it. I committed the final resulting file. Please let me know if there is a different way to fix this.

Reply:
I suspect they are actually dependencies, I tried removing the package from package.json and cleanly installing again using `yarn add` and got the same result.